src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -40,6 +44,16 @@
     private volatile int storageCheckInterval = Integer.MAX_VALUE;
     private final AtomicBoolean resetQuota = new AtomicBoolean(false);
     private final StorageChecker storageChecker = new StorageChecker();
+    private final Gauge limitBytes = KafkaYammerMetrics.defaultRegistry().newGauge(StaticQuotaCallback.class, "LimitBytes", new Gauge<Long>() {
```

Comment:
@tombentley Is the default registry still used? I remember from the Agent work that there was now some other metrics registry used by Kafka, or?